### PR TITLE
docs: cleanup oss installation instructions

### PR DESCRIPTION
**Pull request by MthwRobinson**
### Summary

- Cleans up old installation instructions for `conda` on Windows which are not officially supported.
- Adds notes on system dependencies to the "full installation" section.

### CODE REVIEW

File: open-source/installation/full-installation.mdx [hunk 1]
```diff
@@ -10,6 +10,11 @@ For a complete set of extras catering to every document type, use:
 pip install "unstructured[all-docs]"
 ```
 
+To install `unstructured`, you'll also need to install the following system dependencies: libmagic, poppler-utils, libreoffice, pandoc, and tesseract.
```
Comment (MKhalusova) [inline open-source/installation/full-installation.mdx:13]:
```suggestion
To install `unstructured`, you'll also need to install the following system dependencies: `libmagic`, `poppler-utils`, `libreoffice`, `pandoc`, and `tesseract`.
```

Comment (MKhalusova) [reply open-source/installation/full-installation.mdx:13]: Should we also give links to these libraries?

Comment (MthwRobinson) [reply open-source/installation/full-installation.mdx]: Links added

**Review by MKhalusova — APPROVED** [review]
Thank you for the updates! Looks good to me :)

**Pull request merged**